Merge/mainnet-readiness.md

```diff
@@ -84,6 +86,7 @@ This document outlines various tasks to work through to make the Merge ready for
     * [x] Inherit all prior unit tests and generators
     * [ ] [IN [PROGRESS](https://github.com/ethereum/eth2.0-specs/tree/dev/tests/core/pyspec/eth2spec/test/merge)] Merge specific tests with mocked execution-layer
     * [ ] [IN [PROGRESS](https://github.com/ethereum/consensus-specs/tree/dev/tests/core/pyspec/eth2spec/test/merge/fork_choice)] Fork and fork-choice tests across merge boundary
+    * [ ] Weak subjectivity checkpoint sync readiness
```

Comment:
It's a bit unclear to me what could be tested in the spec with respect to WS. Will any part of WS logic be included in the executable spec?

Reply:
I intended this to mean that clients should test WS safety calculations against [test vectors from the spec (WIP)](https://github.com/ethereum/consensus-specs/pull/2600#issuecomment-926323928).

This item is also not strictly related to the merge specs, but I think it's necessary for merge readiness of clients. Is there a better place for this item in the doc/should it be removed from here?